bootstrap.sh

```diff
@@ -19,19 +20,19 @@ getLocale() {
 
 if [ "$1" == "--force" ]; then
     rm -rf firefoxios-l10n
-    rm -rf ios-l10n-scripts
+    rm -rf LocalizationTools
 fi
 
 if [ "$1" == "--importLocales" ]; then
   # Import locales
-  if [ -d "/firefoxios-l10n" ] && [ -d "/ios-l10n-scripts" ]; then
+  if [ -d "/firefoxios-l10n" ] && [ -d "/LocalizationTools" ]; then
       echo "l10n directories found. Not downloading scripts."
   else
       echo "l10n directory not found. Downloading repo and scripts."
       getLocale
   fi
 
-  ./ios-l10n-scripts/ios-l10n-tools --project-path Client.xcodeproj --l10n-project-path ./firefoxios-l10n --import
+  ./import-strings.sh
```

Comment:
I'm wondering, would it make sense to have the import locales steps entirely in another script file? Instead of calling `sh bootstrap.sh --importLocales` we could just call `sh import-strings.sh` ?

Reply:
I did not want to completely modify the way it was working before, so I tried to leave what is working and add the functionality we were missing. But I can modify that if that would look better

Reply:
Can be done in another task if that's where we wanna go :) Great job on this PR!